diffsims/tests/test_library/test_structure_library.py

```diff
@@ -21,6 +21,17 @@
 
 from diffsims.libraries.structure_library import StructureLibrary
 
+def test_get_library_size():
+    identifiers = ['a', 'b']
+    structures = [1, 2]
+    rotation_list = [[3,1], [4,5]]
```

Comment:
These should be `[[(0,0,0),(0,1,0)],[(0,1,0),(0,2,0)]]` or similar

Reply:
Changed rotation_list to list of tuples. When you say "these" do you mean that the identifiers and structures should be on another form as well?

Reply:
I see now that _each phase_ should have a list of tuples, I will fix this now.

Reply:
orientations/rotation_list is now: [[(0, 0, 0), (0.0, 90.0, -180.0)], [(0, 0, 0)]]